src/lib/storage/chunk.hpp

```diff
@@ -141,6 +141,16 @@ class Chunk : private Noncopyable {
    */
   size_t estimate_memory_usage() const;
 
+  /**
+   * @return A clone of this Chunk, with the same columns, allocator etc.
+   */
+  std::shared_ptr<Chunk> forward_clone() const;
+
+  /**
+   * @return A clone of this Chunk, with the same allocator etc. but all Columns become materialized ValueColumns
+   */
+  std::shared_ptr<Chunk> materialized_clone() const;
```

Comment:
Why not “materialize”? Expensive methods should sound expensive, not like a getter.

Reply:
I think the word `clone` is pretty clear. As an alternative: `copy`?